addons/plugin_refresher/refresh_button.gd

```diff
@@ -214,14 +213,14 @@ func _is_plugin_enabled(plugin_index: int) -> bool:
 func _set_plugin_enabled(plugin_index: int, enabled: bool):
 	refresh_plugin.get_editor_interface().set_plugin_enabled(plugins[plugin_index].directory, enabled)
 
-func _get_editor_setting(name: String, default_value: Variant = null) -> Variant:
```

Comment:
Parse Error: The local function parameter "name" is shadowing an already-declared property at the base class "Node". (Warning treated as error.)